case-studies.markdown

```diff
@@ -15,6 +15,49 @@ figure img { border: 1px solid #999999;}
 figure figcaption { font-style: italic; font-size: 0.85em; text-align: center;}
 </style>
 
+## CarpentryConnects in 2023
+
+### CarpentryConnect New Zealand
+
+Dates: 14 February 2023
+
+Location: Online
+
+Event website: [https://www.eresearchnz2023.org.nz/carpentry-connect/](https://www.eresearchnz2023.org.nz/carpentry-connect/)
+
+For any questions about specific elements around the planning of this event, [please contact eresearchnzconference@auckland.ac.nz](mailto:eresearchnzconference@auckland.ac.nz)
+
+
+## CarpentryConnects in 2021
+
+### CarpentryConnect New Zealand
+
+Dates: 9 February 2021
+
+Location: Wellington, New Zealand and Online
+
+Event website: [https://www.nesi.org.nz/news/2021/02/highlights-carpentry-connect-2021-bringing-together-new-zealands-digital-skills](https://www.nesi.org.nz/news/2021/02/highlights-carpentry-connect-2021-bringing-together-new-zealands-digital-skills)
+
+For any questions about specific elements around the planning of this event, [please contact Megan Guidry](mailto:megan.guidry@nesi.org.nz)
```

Comment:
Megan no longer works here, so I think we might want to remove all contact information from the page unless it is a generic email address.